### PR TITLE
Update ws_desktop_de.ts Translation and fixes.

### PR DESCRIPTION
**Pull request by Evonos**
Added many fixes to the german Translation like MS was translated to "Frau" ( Which means Women ) Guinea pig isnt a term in german to something Experimental so i added "( Experimental )  Alternatively it could be "Versuchskanninchen"  buts "Versuchskanninchen" is a different animal ( Rabbit ) used for laboratory Experimental stuff but we germans usually dont Translate "Named" things and the Experimental builds are named here "Guinea Pig" which would translate Directly ( but not fitting ) to "Meerschweinchen" ( which is just the pet ) and Versuchskanninchen could confuse people because officially its named "guinea pig"

Added / Changed many false Translations.
"Split tunneling" is a named Term and shouldnt be translated ( most vpn handle it that way too )

many small fixes.

Hope my Translation will get added.
also open for feedback. 

Regards
https://new.reddit.com/user/Evonos

### CODE REVIEW

File: client/gui/translations/ws_desktop_de.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@
     </message>
     <message>
         <source>Clear</source>
-        <translation>Klar</translation>
+        <translation>leeren</translation>
```
Comment (Evonos) [inline client/gui/translations/ws_desktop_de.ts:12]: it was "Clear" for klar like a cleaned window but not a emtptied text box / one you can empty with that button.

File: client/gui/translations/ws_desktop_de.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@
     </message>
     <message>
         <source> transferred</source>
-        <translation> überführt</translation>
+        <translation> übertragen</translation>
```
Comment (Evonos) [inline client/gui/translations/ws_desktop_de.ts:62]: "Überführt" is something you do in transport like boxes , prisoners and more.
"Übertragen" is something you do with data , money in digital sense and more its more fitting and correct here.

File: client/gui/translations/ws_desktop_de.ts [hunk 3]
```diff
@@ -71,7 +71,7 @@
     </message>
     <message>
         <source>Connect to rate</source>
-        <translation>Verbinden Sie sich mit dem Tarif</translation>
+        <translation>Verbinden Sie sich zum bewerten</translation>
```
Comment (Evonos) [inline client/gui/translations/ws_desktop_de.ts:74]: Tarif is something like a Mobile contract or even the 10/30gb limits of the free plans
Bewerten = Review / Rate

File: client/gui/translations/ws_desktop_de.ts [hunk 4]
```diff
@@ -92,23 +92,23 @@
     <name>CustomMenuWidget</name>
     <message>
         <source>Undo</source>
-        <translation>Aufmachen</translation>
+        <translation>rückgängig machen</translation>
```
Comment (Evonos) [inline client/gui/translations/ws_desktop_de.ts:95]: "Aufmachen" = Open something in germany like a "Tür Aufmachen" ( Open a Door ) 
Rückgängig machen = Undo

**Comment by jaxu** [comment]
Thank you.  We will review these suggestions and integrate them into a future build.

**Comment by bernerdad** [comment]
Thank you for the PR!

**Pull request merged**